### DISCUSSION TITLE
not signed format files (ps1xml)

### DISCUSSION THREAD

**Discussion by sds95** [pester/Pester#2210] [Q&A] [answered]
Hello! I have no way to downgrade ExecutionPolicy, but otherwise I have no idea how I can import module with files: Pester.Format.ps1xml, PesterConfiguration.Format.ps1xml. Does anyone know how to bypass module import with unsigned files or may be they will be signed in the next pester version?

**Comment by fflaten** [comment, marked as answer]
Thanks for the feedback. Registered it as an issue as they should be signed indeed, hopefully in next release. 🙂
